skywalker/gui.py

```diff
@@ -17,15 +17,13 @@
                               QObject, QEvent)
 from pydm.PyQt.QtGui import QDoubleValidator, QDialog
 
-from pcdsdevices import sim
 from pcdsdevices.epics.attenuator import FeeAtt
```

Comment:
Should we load this from configuration as well?

Reply:
Not sure, it can be a next step. Right now there's a hard-coded simulated config.